src/main/webapp/app/entities/course.model.ts

```diff
@@ -10,6 +10,11 @@ import { LearningGoal } from 'app/entities/learningGoal.model';
 import { Organization } from 'app/entities/organization.model';
 import { Post } from 'app/entities/metis/post.model';
 
+export interface NgxDataEntry {
+    name: string;
+    value: number;
+}
+
```

Comment:
I chose this place in order to make it easily reusable for future charts (as I assumed that the course model might be the "lowest common denominator" for most of the places where charts are needed) without adding more dependencies through additional imports. But if someone comes up with a better place, it would be a pleasure for me to reset it. 😀